src/cli-logger.js

```diff
@@ -7,12 +7,15 @@ const chalk = require("chalk");
 const warn = createLogger("warn", "yellow");
 const error = createLogger("error", "red");
 const debug = createLogger("debug", "blue");
+const log = createLogger("log");
 
 function createLogger(loggerName, color) {
-  const prefix = `[${chalk[color](loggerName)}] `;
-  return function(message) {
+  const prefix = color ? `[${chalk[color](loggerName)}] ` : "";
+  return function(message, opts) {
+    opts = Object.assign({ newline: true }, opts);
     if (shouldLog(loggerName)) {
-      console.error(message.replace(/^/gm, prefix).replace(/[\t ]+$/gm, ""));
```

Comment:
@ikatyang What was this right trim for?

Reply:
It's just my personal preference, not necessary. https://github.com/prettier/prettier/pull/3515#discussion_r157432848